jazz_create-serverless-service/index.js

```diff
@@ -75,24 +76,24 @@ module.exports.handler = (event, context, cb) => {
             })
             .catch(function (err) {
                 logger.error('Error while creating a service : ' + JSON.stringify(err));
-                if(err.jenkins_api_failure) {
-                        serviceDataObject.body = {
-                            "STATUS" : "creation_failed"
-                        };
-                        crud.update(serviceId, serviceDataObject, function (serviceUpdateError, results) {
-                            if (serviceUpdateError) {
-                                var errorMessage = {
-                                    "message": "Error occurred while updating service with failed status.",
-                                    "error" : err
-                                };
-                                return cb(JSON.stringify(errorHandler.throwInternalServerError(errorMessage)));
-                            } else {
-                                logger.error("Updated service catalog with failed status.");
-                                return cb(JSON.stringify(errorHandler.throwInternalServerError(err.message)));
-                            }
-                        });
+                if (err.jenkins_api_failure) {
+                    serviceDataObject.body = {
+                        "STATUS": "creation_failed"
+                    };
+                    crud.update(serviceId, serviceDataObject, function (serviceUpdateError, results) {
```

Comment:
you can use  (serviceUpdateError, results) => here and other places in the code

Reply:
changed